src/libraries/System.ComponentModel.Composition/src/System/ComponentModel/Composition/ReflectionModel/ReflectionComposablePartDefinition.cs

```diff
@@ -154,11 +154,11 @@ public override ComposablePart CreatePart()
             return null;
         }
 
-        internal override bool TryGetExports(ImportDefinition definition, out Tuple<ComposablePartDefinition, ExportDefinition>? singleMatch, out IEnumerable<Tuple<ComposablePartDefinition, ExportDefinition>>? multipleMatches)
+        internal override bool TryGetExports(ImportDefinition definition, [NotNullWhen(true)] out Tuple<ComposablePartDefinition, ExportDefinition>? singleMatch, out IEnumerable<Tuple<ComposablePartDefinition, ExportDefinition>>? multipleMatches)
         {
             if (this.IsGeneric())
             {
-                singleMatch = null;
+                singleMatch = null!; // TODO2
```

Comment:
:memo: the `[NotNullWhen(...)]` annotation on method seems incorrect

Reply:
Ditto